cluster/gce/gci/configure-helper.sh

```diff
@@ -1120,6 +1120,13 @@ function start-rescheduler {
   fi
 }
 
+# Setup working directory for kubelet.
+function setup-kubelet-dir {
+    echo "Making /var/lib/kubelet executable for kubelet"
+    mount --bind /var/lib/kubelet /var/lib/kubelet/
+    mount -B -o remount,exec,suid,dev /var/lib/kubelet    
```

Comment:
Ack. I will clean it up in a separate patch to avoid having to rebase the cherry-pick patch.